x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group2/update.ts

```diff
@@ -174,6 +180,189 @@ export default function createUpdateTests({ getService }: FtrProviderContext) {
+            schedule: { interval: '12s' },
+            actions: [],
+            throttle: '1m',
+            notify_when: 'onThrottleInterval',
+            flapping: {
+              look_back_window: 5,
+              status_change_threshold: 5,
+            },
+          });
+
+        expect(updatedRule.flapping).eql({
+          look_back_window: 5,
+          status_change_threshold: 5,
+        });
+      });
+
+      it('should allow flapping to be removed via update', async () => {
+        const response = await supertest
+          .post(`${getUrlPrefix(Spaces.space1.id)}/api/alerting/rule`)
+          .set('kbn-xsrf', 'foo')
+          .send(
+            getTestRuleData({
+              flapping: {
+                look_back_window: 5,
+                status_change_threshold: 5,
+              },
+            })
+          );
+
+        expect(response.body.flapping).eql({
+          look_back_window: 5,
+          status_change_threshold: 5,
+        });
+
+        objectRemover.add(Spaces.space1.id, response.body.id, 'rule', 'alerting');
+
+        const { body: updatedRule } = await supertest
+          .put(`${getUrlPrefix(Spaces.space1.id)}/api/alerting/rule/${response.body.id}`)
+          .set('kbn-xsrf', 'foo')
+          .send({
+            name: 'bcd',
+            tags: ['foo'],
+            params: {
+              foo: true,
+            },
+            schedule: { interval: '12s' },
+            actions: [],
+            throttle: '1m',
+            notify_when: 'onThrottleInterval',
+            flapping: null,
+          });
+
+        expect(updatedRule.flapping).eql(null);
+      });
+
+      it('should throw if flapping is updated when global flapping is off', async () => {
+        const response = await supertest
+          .post(`${getUrlPrefix(Spaces.space1.id)}/api/alerting/rule`)
+          .set('kbn-xsrf', 'foo')
+          .send(getTestRuleData());
+
+        objectRemover.add(Spaces.space1.id, response.body.id, 'rule', 'alerting');
+
+        await supertest
+          .post(`${getUrlPrefix(Spaces.space1.id)}/internal/alerting/rules/settings/_flapping`)
+          .set('kbn-xsrf', 'foo')
+          .send({
+            enabled: false,
+            look_back_window: 5,
+            status_change_threshold: 5,
+          });
+
+        await supertest
+          .put(`${getUrlPrefix(Spaces.space1.id)}/api/alerting/rule/${response.body.id}`)
+          .set('kbn-xsrf', 'foo')
+          .send({
+            name: 'bcd',
+            tags: ['foo'],
+            params: {
+              foo: true,
+            },
+            schedule: { interval: '12s' },
+            actions: [],
+            throttle: '1m',
+            notify_when: 'onThrottleInterval',
+            flapping: {
+              look_back_window: 5,
+              status_change_threshold: 5,
+            },
+          })
+          .expect(400);
+
+        // Can still update when global flapping is off if not updating flapping
```

Comment:
super nit: What about isolating this test scenario to another test?

Reply:
I have to do it here because it's in a state where this should be tested (global off, rule already has flapping property). If i were to split this i literally just copy this entire test but give it a different name.